.github/workflows/lighthouse-ci-workflow.yml

```diff
@@ -4,14 +4,14 @@ name: Lighthouse CI
 # or reopened and on master so comparisons to ancestors can be made.
 on:
   push:
-    branches: 
+    branches:
       - master
   pull_request:
     paths:
       - '**.js'
       - '!src/site/_data/contributors.js'
       - '!src/site/_data/countries.js'
-      - '!src/site/_data/postTags.js'
+      - '!src/site/_data/tagsData.json'
```

Comment:
I think you can remove this line entirely. The line at the top is telling it to include only .js files. Since this is now a .json file it should be ignored.